charts/extended-daemon-set/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: v0.6.0
+description: Extended Daemonset Controller
+name: extendeddaemonset
+version: v0.6.0
```

Comment:
Should we start with 0.6.0 to be consistent with the appVersion? Or should we set this to 0.1.0 ?

Reply:
I believe we should start with 0.1.0, the chart versions and the app versions don't have to be aligned (and won't be anyway

Reply:
Changed :+1: